acceptance: TestDockerCLI_test_distinguished_name_validation failed

acceptance.TestDockerCLI_test_distinguished_name_validation [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14764274?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14764274?buildTab=artifacts#/) on master @ [392c924bdde20dadfa57fcd3c25e9fdf7554f0a7](https://github.com/cockroachdb/cockroach/commits/392c924bdde20dadfa57fcd3c25e9fdf7554f0a7):


```
=== RUN   TestDockerCLI_test_distinguished_name_validation
    test_log_scope.go:170: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation1210780389
    test_log_scope.go:81: use -show-logs to present logs inline
    generated_cli_test.go:180: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation1210780389
--- FAIL: TestDockerCLI_test_distinguished_name_validation (64.11s)
=== RUN   TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl
    util_docker.go:259: non-zero exit code: 1
    --- FAIL: TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl (57.37s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI_test_distinguished_name_validation.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>


Jira issue: CRDB-37669

acceptance.TestDockerCLI_test_distinguished_name_validation [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14765231?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14765231?buildTab=artifacts#/) on master @ [54b2e886b5f79aec7632e395f06929d4c8f5b887](https://github.com/cockroachdb/cockroach/commits/54b2e886b5f79aec7632e395f06929d4c8f5b887):


```
=== RUN   TestDockerCLI_test_distinguished_name_validation
    test_log_scope.go:170: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation3379032720
    test_log_scope.go:81: use -show-logs to present logs inline
    generated_cli_test.go:180: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation3379032720
--- FAIL: TestDockerCLI_test_distinguished_name_validation (63.52s)
=== RUN   TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl
    util_docker.go:259: non-zero exit code: 1
    --- FAIL: TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl (56.87s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI_test_distinguished_name_validation.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

acceptance.TestDockerCLI_test_distinguished_name_validation [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14766013?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14766013?buildTab=artifacts#/) on master @ [e9d1b467fbe5a2e6751f4eaf39d95c43f7c89068](https://github.com/cockroachdb/cockroach/commits/e9d1b467fbe5a2e6751f4eaf39d95c43f7c89068):


```
=== RUN   TestDockerCLI_test_distinguished_name_validation
    test_log_scope.go:170: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation3031474528
    test_log_scope.go:81: use -show-logs to present logs inline
    generated_cli_test.go:180: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation3031474528
--- FAIL: TestDockerCLI_test_distinguished_name_validation (61.07s)
=== RUN   TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl
    util_docker.go:259: non-zero exit code: 1
    --- FAIL: TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl (55.46s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #122036 acceptance: TestDockerCLI_test_distinguished_name_validation failed [C-test-failure O-robot T-sql-foundations branch-release-24.1 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI_test_distinguished_name_validation.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

acceptance.TestDockerCLI_test_distinguished_name_validation [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14766855?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14766855?buildTab=artifacts#/) on master @ [64341d7802a1ba8b3fbd6a59e5afd58b693dc569](https://github.com/cockroachdb/cockroach/commits/64341d7802a1ba8b3fbd6a59e5afd58b693dc569):


```
=== RUN   TestDockerCLI_test_distinguished_name_validation
    test_log_scope.go:170: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation1970697440
    test_log_scope.go:81: use -show-logs to present logs inline
    generated_cli_test.go:180: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation1970697440
--- FAIL: TestDockerCLI_test_distinguished_name_validation (62.20s)
=== RUN   TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl
    util_docker.go:259: non-zero exit code: 1
    --- FAIL: TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl (55.56s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #122036 acceptance: TestDockerCLI_test_distinguished_name_validation failed [C-test-failure O-robot T-sql-foundations branch-release-24.1 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI_test_distinguished_name_validation.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

acceptance.TestDockerCLI_test_distinguished_name_validation [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14767764?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14767764?buildTab=artifacts#/) on master @ [64341d7802a1ba8b3fbd6a59e5afd58b693dc569](https://github.com/cockroachdb/cockroach/commits/64341d7802a1ba8b3fbd6a59e5afd58b693dc569):


```
=== RUN   TestDockerCLI_test_distinguished_name_validation
    test_log_scope.go:170: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation1332629939
    test_log_scope.go:81: use -show-logs to present logs inline
    generated_cli_test.go:180: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation1332629939
--- FAIL: TestDockerCLI_test_distinguished_name_validation (65.27s)
=== RUN   TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl
    util_docker.go:259: non-zero exit code: 1
    --- FAIL: TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl (58.12s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #122036 acceptance: TestDockerCLI_test_distinguished_name_validation failed [C-test-failure O-robot T-sql-foundations branch-release-24.1 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI_test_distinguished_name_validation.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

acceptance.TestDockerCLI_test_distinguished_name_validation [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14767910?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14767910?buildTab=artifacts#/) on master @ [bb71f2810ad27604a5b77b69183c6e5d025f449e](https://github.com/cockroachdb/cockroach/commits/bb71f2810ad27604a5b77b69183c6e5d025f449e):


```
=== RUN   TestDockerCLI_test_distinguished_name_validation
    test_log_scope.go:170: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation626226452
    test_log_scope.go:81: use -show-logs to present logs inline
    generated_cli_test.go:180: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation626226452
--- FAIL: TestDockerCLI_test_distinguished_name_validation (61.75s)
=== RUN   TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl
    util_docker.go:259: non-zero exit code: 1
    --- FAIL: TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl (55.70s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #122036 acceptance: TestDockerCLI_test_distinguished_name_validation failed [C-test-failure O-robot T-sql-foundations branch-release-24.1 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI_test_distinguished_name_validation.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

acceptance.TestDockerCLI_test_distinguished_name_validation [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14769648?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14769648?buildTab=artifacts#/) on master @ [dc750754297a9bdf237aefd344765aa4a5992fef](https://github.com/cockroachdb/cockroach/commits/dc750754297a9bdf237aefd344765aa4a5992fef):


```
=== RUN   TestDockerCLI_test_distinguished_name_validation
    test_log_scope.go:170: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation2804398985
    test_log_scope.go:81: use -show-logs to present logs inline
    generated_cli_test.go:180: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation2804398985
--- FAIL: TestDockerCLI_test_distinguished_name_validation (63.43s)
=== RUN   TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl
    util_docker.go:259: non-zero exit code: 1
    --- FAIL: TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl (57.50s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #122036 acceptance: TestDockerCLI_test_distinguished_name_validation failed [C-test-failure O-robot P-1 T-sql-foundations branch-release-24.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI_test_distinguished_name_validation.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

acceptance.TestDockerCLI_test_distinguished_name_validation [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14769712?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14769712?buildTab=artifacts#/) on master @ [dc750754297a9bdf237aefd344765aa4a5992fef](https://github.com/cockroachdb/cockroach/commits/dc750754297a9bdf237aefd344765aa4a5992fef):


```
=== RUN   TestDockerCLI_test_distinguished_name_validation
    test_log_scope.go:170: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation719374189
    test_log_scope.go:81: use -show-logs to present logs inline
    generated_cli_test.go:180: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation719374189
--- FAIL: TestDockerCLI_test_distinguished_name_validation (54.07s)
=== RUN   TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl
    util_docker.go:259: non-zero exit code: 1
    --- FAIL: TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl (51.50s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #122036 acceptance: TestDockerCLI_test_distinguished_name_validation failed [C-test-failure O-robot P-1 T-sql-foundations branch-release-24.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI_test_distinguished_name_validation.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

acceptance.TestDockerCLI_test_distinguished_name_validation [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14769915?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14769915?buildTab=artifacts#/) on master @ [8ff9f39a5185b55a438e23fe36a1397ccf4df63a](https://github.com/cockroachdb/cockroach/commits/8ff9f39a5185b55a438e23fe36a1397ccf4df63a):


```
=== RUN   TestDockerCLI_test_distinguished_name_validation
    test_log_scope.go:170: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation2360038509
    test_log_scope.go:81: use -show-logs to present logs inline
    generated_cli_test.go:180: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation2360038509
--- FAIL: TestDockerCLI_test_distinguished_name_validation (63.92s)
=== RUN   TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl
    util_docker.go:259: non-zero exit code: 1
    --- FAIL: TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl (57.34s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #122036 acceptance: TestDockerCLI_test_distinguished_name_validation failed [C-test-failure O-robot P-1 T-sql-foundations branch-release-24.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI_test_distinguished_name_validation.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

acceptance.TestDockerCLI_test_distinguished_name_validation [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14770181?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14770181?buildTab=artifacts#/) on master @ [f9ab8e3c2ad0d38f0fbcea66e483344df5fcc6db](https://github.com/cockroachdb/cockroach/commits/f9ab8e3c2ad0d38f0fbcea66e483344df5fcc6db):


```
=== RUN   TestDockerCLI_test_distinguished_name_validation
    test_log_scope.go:170: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation968887056
    test_log_scope.go:81: use -show-logs to present logs inline
    generated_cli_test.go:180: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation968887056
--- FAIL: TestDockerCLI_test_distinguished_name_validation (60.35s)
=== RUN   TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl
    util_docker.go:259: non-zero exit code: 1
    --- FAIL: TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl (54.36s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #122036 acceptance: TestDockerCLI_test_distinguished_name_validation failed [C-test-failure O-robot P-1 T-sql-foundations branch-release-24.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI_test_distinguished_name_validation.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

acceptance.TestDockerCLI_test_distinguished_name_validation [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14770586?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14770586?buildTab=artifacts#/) on master @ [c277d95753ebc97a3bd4d55d6cd1faf452e70ed2](https://github.com/cockroachdb/cockroach/commits/c277d95753ebc97a3bd4d55d6cd1faf452e70ed2):


```
=== RUN   TestDockerCLI_test_distinguished_name_validation
    test_log_scope.go:170: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation2591519014
    test_log_scope.go:81: use -show-logs to present logs inline
    generated_cli_test.go:180: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation2591519014
--- FAIL: TestDockerCLI_test_distinguished_name_validation (61.51s)
=== RUN   TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl
    util_docker.go:259: non-zero exit code: 1
    --- FAIL: TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl (54.19s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #122036 acceptance: TestDockerCLI_test_distinguished_name_validation failed [C-test-failure O-robot P-1 T-sql-foundations branch-release-24.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI_test_distinguished_name_validation.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

acceptance.TestDockerCLI_test_distinguished_name_validation [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14770986?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14770986?buildTab=artifacts#/) on master @ [bbf8beadcb82149db5cab7d8ba56969da192a237](https://github.com/cockroachdb/cockroach/commits/bbf8beadcb82149db5cab7d8ba56969da192a237):


```
=== RUN   TestDockerCLI_test_distinguished_name_validation
    test_log_scope.go:170: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation2776327789
    test_log_scope.go:81: use -show-logs to present logs inline
    generated_cli_test.go:180: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation2776327789
--- FAIL: TestDockerCLI_test_distinguished_name_validation (63.13s)
=== RUN   TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl
    util_docker.go:259: non-zero exit code: 1
    --- FAIL: TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl (56.17s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #122036 acceptance: TestDockerCLI_test_distinguished_name_validation failed [C-test-failure O-robot P-1 T-sql-foundations branch-release-24.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI_test_distinguished_name_validation.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

acceptance.TestDockerCLI_test_distinguished_name_validation [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14771425?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14771425?buildTab=artifacts#/) on master @ [3ebc36eaa2fb9cbac299f1d92a9cf7da7f2033bd](https://github.com/cockroachdb/cockroach/commits/3ebc36eaa2fb9cbac299f1d92a9cf7da7f2033bd):


```
=== RUN   TestDockerCLI_test_distinguished_name_validation
    test_log_scope.go:170: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation1864218032
    test_log_scope.go:81: use -show-logs to present logs inline
    generated_cli_test.go:180: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation1864218032
--- FAIL: TestDockerCLI_test_distinguished_name_validation (62.30s)
=== RUN   TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl
    util_docker.go:259: non-zero exit code: 1
    --- FAIL: TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl (55.77s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #122036 acceptance: TestDockerCLI_test_distinguished_name_validation failed [C-test-failure O-robot P-1 T-sql-foundations branch-release-24.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI_test_distinguished_name_validation.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

acceptance.TestDockerCLI_test_distinguished_name_validation [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14771624?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14771624?buildTab=artifacts#/) on master @ [3ebc36eaa2fb9cbac299f1d92a9cf7da7f2033bd](https://github.com/cockroachdb/cockroach/commits/3ebc36eaa2fb9cbac299f1d92a9cf7da7f2033bd):


```
=== RUN   TestDockerCLI_test_distinguished_name_validation
    test_log_scope.go:170: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation3491162577
    test_log_scope.go:81: use -show-logs to present logs inline
    generated_cli_test.go:180: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation3491162577
--- FAIL: TestDockerCLI_test_distinguished_name_validation (63.17s)
=== RUN   TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl
    util_docker.go:259: non-zero exit code: 1
    --- FAIL: TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl (56.98s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #122036 acceptance: TestDockerCLI_test_distinguished_name_validation failed [C-test-failure O-robot P-1 T-sql-foundations branch-release-24.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI_test_distinguished_name_validation.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

acceptance.TestDockerCLI_test_distinguished_name_validation [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14771652?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14771652?buildTab=artifacts#/) on master @ [3ebc36eaa2fb9cbac299f1d92a9cf7da7f2033bd](https://github.com/cockroachdb/cockroach/commits/3ebc36eaa2fb9cbac299f1d92a9cf7da7f2033bd):


```
=== RUN   TestDockerCLI_test_distinguished_name_validation
    test_log_scope.go:170: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation1783858160
    test_log_scope.go:81: use -show-logs to present logs inline
    generated_cli_test.go:180: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation1783858160
--- FAIL: TestDockerCLI_test_distinguished_name_validation (54.06s)
=== RUN   TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl
    util_docker.go:259: non-zero exit code: 1
    --- FAIL: TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl (51.56s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #122036 acceptance: TestDockerCLI_test_distinguished_name_validation failed [C-test-failure O-robot P-1 T-sql-foundations branch-release-24.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI_test_distinguished_name_validation.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

acceptance.TestDockerCLI_test_distinguished_name_validation [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14772047?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14772047?buildTab=artifacts#/) on master @ [911f1ce389342459592d89fe7b78bcba3a2d265a](https://github.com/cockroachdb/cockroach/commits/911f1ce389342459592d89fe7b78bcba3a2d265a):


```
=== RUN   TestDockerCLI_test_distinguished_name_validation
    test_log_scope.go:170: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation4173290673
    test_log_scope.go:81: use -show-logs to present logs inline
    generated_cli_test.go:180: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation4173290673
--- FAIL: TestDockerCLI_test_distinguished_name_validation (62.16s)
=== RUN   TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl
    util_docker.go:259: non-zero exit code: 1
    --- FAIL: TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl (55.02s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #122036 acceptance: TestDockerCLI_test_distinguished_name_validation failed [C-test-failure O-robot P-1 T-sql-foundations branch-release-24.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI_test_distinguished_name_validation.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

acceptance.TestDockerCLI_test_distinguished_name_validation [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14772227?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14772227?buildTab=artifacts#/) on master @ [911f1ce389342459592d89fe7b78bcba3a2d265a](https://github.com/cockroachdb/cockroach/commits/911f1ce389342459592d89fe7b78bcba3a2d265a):


```
=== RUN   TestDockerCLI_test_distinguished_name_validation
    test_log_scope.go:170: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation4150492478
    test_log_scope.go:81: use -show-logs to present logs inline
    generated_cli_test.go:180: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation4150492478
--- FAIL: TestDockerCLI_test_distinguished_name_validation (61.58s)
=== RUN   TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl
    util_docker.go:259: non-zero exit code: 1
    --- FAIL: TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl (55.46s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #122036 acceptance: TestDockerCLI_test_distinguished_name_validation failed [C-test-failure O-robot P-1 T-sql-foundations branch-release-24.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI_test_distinguished_name_validation.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

acceptance.TestDockerCLI_test_distinguished_name_validation [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14772369?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14772369?buildTab=artifacts#/) on master @ [911f1ce389342459592d89fe7b78bcba3a2d265a](https://github.com/cockroachdb/cockroach/commits/911f1ce389342459592d89fe7b78bcba3a2d265a):


```
=== RUN   TestDockerCLI_test_distinguished_name_validation
    test_log_scope.go:170: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation2742862619
    test_log_scope.go:81: use -show-logs to present logs inline
    generated_cli_test.go:180: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation2742862619
--- FAIL: TestDockerCLI_test_distinguished_name_validation (54.48s)
=== RUN   TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl
    util_docker.go:259: non-zero exit code: 1
    --- FAIL: TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl (52.03s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #122036 acceptance: TestDockerCLI_test_distinguished_name_validation failed [C-test-failure O-robot P-1 T-sql-foundations branch-release-24.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI_test_distinguished_name_validation.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

acceptance.TestDockerCLI_test_distinguished_name_validation [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14772892?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14772892?buildTab=artifacts#/) on master @ [911f1ce389342459592d89fe7b78bcba3a2d265a](https://github.com/cockroachdb/cockroach/commits/911f1ce389342459592d89fe7b78bcba3a2d265a):


```
=== RUN   TestDockerCLI_test_distinguished_name_validation
    test_log_scope.go:170: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation3936927558
    test_log_scope.go:81: use -show-logs to present logs inline
    generated_cli_test.go:180: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation3936927558
--- FAIL: TestDockerCLI_test_distinguished_name_validation (64.90s)
=== RUN   TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl
    util_docker.go:259: non-zero exit code: 1
    --- FAIL: TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl (58.30s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #122036 acceptance: TestDockerCLI_test_distinguished_name_validation failed [C-test-failure O-robot P-1 T-sql-foundations branch-release-24.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI_test_distinguished_name_validation.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

acceptance.TestDockerCLI_test_distinguished_name_validation [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14773241?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14773241?buildTab=artifacts#/) on master @ [911f1ce389342459592d89fe7b78bcba3a2d265a](https://github.com/cockroachdb/cockroach/commits/911f1ce389342459592d89fe7b78bcba3a2d265a):


```
=== RUN   TestDockerCLI_test_distinguished_name_validation
    test_log_scope.go:170: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation3577203957
    test_log_scope.go:81: use -show-logs to present logs inline
    generated_cli_test.go:180: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation3577203957
--- FAIL: TestDockerCLI_test_distinguished_name_validation (53.74s)
=== RUN   TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl
    util_docker.go:259: non-zero exit code: 1
    --- FAIL: TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl (51.32s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #122036 acceptance: TestDockerCLI_test_distinguished_name_validation failed [C-test-failure O-robot P-1 T-sql-foundations branch-release-24.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI_test_distinguished_name_validation.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

acceptance.TestDockerCLI_test_distinguished_name_validation [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14773250?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14773250?buildTab=artifacts#/) on master @ [911f1ce389342459592d89fe7b78bcba3a2d265a](https://github.com/cockroachdb/cockroach/commits/911f1ce389342459592d89fe7b78bcba3a2d265a):


```
=== RUN   TestDockerCLI_test_distinguished_name_validation
    test_log_scope.go:170: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation1246081199
    test_log_scope.go:81: use -show-logs to present logs inline
    generated_cli_test.go:180: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation1246081199
--- FAIL: TestDockerCLI_test_distinguished_name_validation (53.73s)
=== RUN   TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl
    util_docker.go:259: non-zero exit code: 1
    --- FAIL: TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl (51.23s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #122036 acceptance: TestDockerCLI_test_distinguished_name_validation failed [C-test-failure O-robot P-1 T-sql-foundations branch-release-24.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI_test_distinguished_name_validation.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

acceptance.TestDockerCLI_test_distinguished_name_validation [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14778252?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14778252?buildTab=artifacts#/) on master @ [e078ee499dd4474fb549aaf3ed6c0647640be55c](https://github.com/cockroachdb/cockroach/commits/e078ee499dd4474fb549aaf3ed6c0647640be55c):


```
=== RUN   TestDockerCLI_test_distinguished_name_validation
    test_log_scope.go:170: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation1496292094
    test_log_scope.go:81: use -show-logs to present logs inline
    generated_cli_test.go:180: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation1496292094
--- FAIL: TestDockerCLI_test_distinguished_name_validation (61.98s)
=== RUN   TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl
    util_docker.go:259: non-zero exit code: 1
    --- FAIL: TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl (56.12s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #122036 acceptance: TestDockerCLI_test_distinguished_name_validation failed [C-test-failure O-robot P-1 T-sql-foundations branch-release-24.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI_test_distinguished_name_validation.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

acceptance.TestDockerCLI_test_distinguished_name_validation [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14778879?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14778879?buildTab=artifacts#/) on master @ [4683d652f4bf072fddc9f426d0291c387fa17cc8](https://github.com/cockroachdb/cockroach/commits/4683d652f4bf072fddc9f426d0291c387fa17cc8):


```
=== RUN   TestDockerCLI_test_distinguished_name_validation
    test_log_scope.go:170: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation981079319
    test_log_scope.go:81: use -show-logs to present logs inline
    generated_cli_test.go:180: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation981079319
--- FAIL: TestDockerCLI_test_distinguished_name_validation (65.13s)
=== RUN   TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl
    util_docker.go:259: non-zero exit code: 1
    --- FAIL: TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl (56.93s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #122036 acceptance: TestDockerCLI_test_distinguished_name_validation failed [C-test-failure O-robot P-1 T-sql-foundations branch-release-24.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI_test_distinguished_name_validation.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

acceptance.TestDockerCLI_test_distinguished_name_validation [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14779895?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14779895?buildTab=artifacts#/) on master @ [4683d652f4bf072fddc9f426d0291c387fa17cc8](https://github.com/cockroachdb/cockroach/commits/4683d652f4bf072fddc9f426d0291c387fa17cc8):


```
=== RUN   TestDockerCLI_test_distinguished_name_validation
    test_log_scope.go:170: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation1847401177
    test_log_scope.go:81: use -show-logs to present logs inline
    generated_cli_test.go:180: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation1847401177
--- FAIL: TestDockerCLI_test_distinguished_name_validation (63.89s)
=== RUN   TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl
    util_docker.go:259: non-zero exit code: 1
    --- FAIL: TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl (57.54s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #122036 acceptance: TestDockerCLI_test_distinguished_name_validation failed [C-test-failure O-robot P-1 T-sql-foundations branch-release-24.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI_test_distinguished_name_validation.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

acceptance.TestDockerCLI_test_distinguished_name_validation [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14780268?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14780268?buildTab=artifacts#/) on master @ [4683d652f4bf072fddc9f426d0291c387fa17cc8](https://github.com/cockroachdb/cockroach/commits/4683d652f4bf072fddc9f426d0291c387fa17cc8):


```
=== RUN   TestDockerCLI_test_distinguished_name_validation
    test_log_scope.go:170: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation323082179
    test_log_scope.go:81: use -show-logs to present logs inline
    generated_cli_test.go:180: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation323082179
--- FAIL: TestDockerCLI_test_distinguished_name_validation (62.05s)
=== RUN   TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl
    util_docker.go:259: non-zero exit code: 1
    --- FAIL: TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl (56.23s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #122036 acceptance: TestDockerCLI_test_distinguished_name_validation failed [C-test-failure O-robot P-1 T-sql-foundations branch-release-24.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI_test_distinguished_name_validation.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

acceptance.TestDockerCLI_test_distinguished_name_validation [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14781448?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/14781448?buildTab=artifacts#/) on master @ [4686438a9170d29147c94502e4a44521376fd212](https://github.com/cockroachdb/cockroach/commits/4686438a9170d29147c94502e4a44521376fd212):


```
=== RUN   TestDockerCLI_test_distinguished_name_validation
    test_log_scope.go:170: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation3183476333
    test_log_scope.go:81: use -show-logs to present logs inline
    generated_cli_test.go:180: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/4bca97a9a370e463f46a2a048feaf3a0/logTestDockerCLI_test_distinguished_name_validation3183476333
--- FAIL: TestDockerCLI_test_distinguished_name_validation (64.03s)
=== RUN   TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl
    util_docker.go:259: non-zero exit code: 1
    --- FAIL: TestDockerCLI_test_distinguished_name_validation/test_distinguished_name_validation.tcl (57.20s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #122036 acceptance: TestDockerCLI_test_distinguished_name_validation failed [C-test-failure O-robot P-1 T-sql-foundations branch-release-24.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI_test_distinguished_name_validation.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

--- a/build/teamcity/cockroach/ci/tests-aws-linux-arm64/acceptance.sh
+++ b/build/teamcity/cockroach/ci/tests-aws-linux-arm64/acceptance.sh
@@ -33,6 +33,7 @@ $BAZCI --artifacts_dir=$PWD/artifacts -- \
   "--test_tmpdir=$ARTIFACTSDIR" \
   --test_arg=-l="$ARTIFACTSDIR" \
   --test_arg=-b=$PWD/artifacts/cockroach \
+  --test_env=COCKROACH_DEV_LICENSE  \
   --test_env=TZ=America/New_York \
   --profile=$PWD/artifacts/profile.gz \
   --test_timeout=1800 || status=$?

